errors/index.js

```diff
@@ -0,0 +1,19 @@
+exports.handleNonexistentEndpoint = (req, res, next) => {
+    res.status(404).send({ msg: 'Not found' });
+}
+
+exports.handleCustomErrors = (err, req, res, next) => {
+    console.log(err);
+    next(err);
+}
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+    console.log(err);
+    next(err);
+}
```

Comment:
Worth noting these aren't really necessary at this point in the server, because you've not actually got any endpoints that can cause custom errors or psql errors yet